com.microsoft.mrtk.input/Utilities/PoseSource/PinchPoseSource.cs

```diff
@@ -12,47 +11,25 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// <summary>
     /// A pose source which gets the pinch pose of a specific hand
     /// </summary>
-    public class PinchPoseSource : IPoseSource
+    public class PinchPoseSource : HandBasedPoseSource
     {
-        private HandsAggregatorSubsystem handsAggregator;
-
         /// <summary>
-        /// Cached reference to hands aggregator for efficient per-frame use.
+        /// Tries to get the pinch pose of a specific hand.
         /// </summary>
-        protected HandsAggregatorSubsystem HandsAggregator => handsAggregator ??= HandsUtils.GetSubsystem();
-
-        [SerializeField]
-        [Tooltip("The hand on which to track the joint.")]
-        private Handedness hand;
-
-        /// <summary>
-        /// The hand on which to track the joint.
-        /// </summary>
-        internal Handedness Hand { get => hand; set => hand = value; }
-
-        /// <summary>
-        /// A cache of the hand joint pose returned by the hands aggregator
-        /// </summary>
-        private HandJointPose cachedPinchPose;
```

Comment:
Just wondering what the performance benefits/drawbacks are to caching the HandJointPose here and keeping it on the Heap vs. generating one of these structs on the stack each time this method is called.

Good with either approach!

Reply:
I did some measurements, and the perf was pretty much identical. I had thought there might be some improvement to the new approach, but it ended up being a no-op!